string where
geometry_msgs/Pose pose